python/cudf/cudf/core/column/string.py

```diff
@@ -620,12 +635,20 @@ def contains(
         elif na is not np.nan:
             raise NotImplementedError("`na` parameter is not yet supported")
 
-        return self._return_or_inplace(
-            cpp_contains_re(self._column, pat)
-            if regex is True
-            else cpp_contains(self._column, as_scalar(pat, "str")),
-            **kwargs,
-        )
+        if pat is None:
+            result_col = column.column_empty(
+                len(self._column), dtype="bool", masked=True
+            )
```

Comment:
Should we leave this up to libcudf, or does the python behavior differ from what's expected from libcudf?

Reply:
This logic was already there. It is likely to be consistent with some Pandas or Python feature.

Reply:
Other examples that I referenced for this:
https://github.com/rapidsai/cudf/blob/bdba041fa6e0bdfaf56a97af4d4c391264986e2c/python/cudf/cudf/core/column/string.py#L3143
https://github.com/rapidsai/cudf/blob/bdba041fa6e0bdfaf56a97af4d4c391264986e2c/python/cudf/cudf/core/column/string.py#L3208